ops/terraform/services/static-site/r53.tf

```diff
@@ -0,0 +1,13 @@
+# resource "aws_route53_record" "static_env" {
```

Comment:
Is all the commented out stuff in here intended to be included in the future?

Reply:
If we acquire signed certificates, that file will be uncommented.

Reply:
Can we add a comment at the top of this file to clarify that? Just makes it easier to ensure we want to keep this around and it wasn't intended to be removed.